arm/comm.go

```diff
@@ -420,19 +421,21 @@ func (x *xArm) MoveThroughJointPositions(
 			"max acceleration",
 		)
 	}
-	// Ensure the robot is in correct mode to move
+	// set to servo motion mode
+	if err := x.setMotionMode(ctx, servoMotionMode); err != nil {
+		return err
+	}
+
```

Comment:
Do we need this for the other arm moving APIs? MoveToPosition, MoveToJointPositions? Or are they all calling start?